setup.py

```diff
@@ -12,7 +12,7 @@ def readme():
 
 setup(
     name="gestalt-cfg",
-    version="3.4.2",
+    version="3.4.2beta11",
```

Comment:
```suggestion
    version="3.4.2",
```
should this be changed to just 3.4.2 before merging?

Reply:
Yup, missed that one.

Reply:
Done